config/checkstyle/checkstyle.xml

```diff
@@ -112,7 +112,7 @@
             <property name="sortImportsInGroupAlphabetically" value="true"/>
         </module>
         <module name="CyclomaticComplexity">
```

Comment:
We should just remove this rule. It's not the sort of thing an automated tool can judge. Much better to judge it in a pull request.